.changeset/selfish-years-think.md

```diff
@@ -0,0 +1,7 @@
+---
+"@comet/brevo-api": patch
+---
+
+Remove `scope` from `BrevoConfig` database migration in module
+
+Add database migration for adding individual `scope` columns to `BrevoConfig` instead.
```

Comment:
```suggestion
A custom database migration must be created in the project to add individual scope columns to BrevoConfig.
```

Maybe like that? So it's clear that this is something that needs to be done in the project.